### ISSUE TITLE
i cant run it locally 

### ISSUE THREAD

**Issue by shxrif** [velopert/veltrends#28]
i tried using the postgres, and the frontend works like i tried to go to the login page, but in the home page it is showing some api error 500

**Comment by solomonshalom** [comment]
Have you tried to deploy and set the right credentials for veltrends-server and have you then connected it to the frontend?

**Comment by shxrif** [comment]
nope can u guide me in this stuff thanks

**Comment by solomonshalom** [comment]
Sure!

I haven't particularly used the code base before but I believe what I would do:

1. Clone the entire repository in your Git account (Not locally)

2. Using a hosting solution such as Vercel or Netlify, import the repository and deploy the `veltrends-server` folder first and fill in the env's

3. Then, host the `veltrends-web` folder and fill in the env

*Remember that you can do the exact same thing locally, just don't forget to install the dependencies for the entire repository and then install and run the folders separately

**Comment by dexjr** [comment]
if u could run the backend in a codespace and send the link would be nice

**Comment by shxrif** [comment]
lets connect on telegram(https://t.me/ayxxn_shxrif)  for faster change of text... and if that goes nice ... will add simple bash script to automate that stuff for others

**Comment by solomonshalom** [comment]
OH, am not on telegram but am on discord! :D

**Comment by shxrif** [comment]
yeah please share ur discord acc or mine username is "ayaansharif"
